### ISSUE TITLE
No network connectivity when running inside nsntrace

### ISSUE THREAD

**Issue by edi9999** [nsntrace/nsntrace#22]
When I run : 

```
sudo tcpdump -w my.pcap -i wlp2s0
wget www.google.com
```

I see network requests when opening my.pcap in wireshark.

However, when running 

```
sudo ./nsntrace -d wlp2s0 wget www.google.com
```

I get : 

```
sudo ./nsntrace -d wlp2s0 wget www.google.com
Starting network trace of 'wget' on interface wlp2s0.
Your IP address in this trace is 172.16.42.255.
Use ctrl-c to end at any time.

--2017-06-29 12:30:03--  http://www.google.com/
Resolving www.google.com (www.google.com)...


failed: Name or service not known.
wget: unable to resolve host address ‘www.google.com’
```

Any idea what is happening ?

**Comment by edi9999** [comment]
If I run : 

```
sudo ./nsntrace -d wlp2s0 ip a
Starting network trace of 'ip' on interface wlp2s0.
Your IP address in this trace is 172.16.42.255.
Use ctrl-c to end at any time.

1: lo: <LOOPBACK> mtu 65536 qdisc noop state DOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: nsntrace-netns@if152: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether ce:b7:f4:31:0d:53 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 172.16.42.255/31 scope global nsntrace-netns
       valid_lft forever preferred_lft forever
    inet6 fe80::ccb7:f4ff:fe31:d53/64 scope link tentative
       valid_lft forever preferred_lft forever
```

**Comment by mgedmin** [comment]
What's in your `/etc/resolv.conf`?  I'm having this problem when nsntrace is trying to talk to 127.0.0.53 and is getting no response.

**Comment by jonasdn** [comment]
Applications such as systemd-resolv or dnsmasq will sometime run nameserver on loopback addresses (127.x.x.x) and that will not work from the network namespace nsntrace creates.

This is addressed with #33 which is merged in release 4. An `--use-public-dns` option is added that uses nameservers from Quad9, OpenDNS, Google and Cloudflare.
